buildSrc/settings.gradle.kts

```diff
@@ -4,4 +4,10 @@ dependencyResolutionManagement {
             from(files("../gradle/libs.versions.toml"))
         }
     }
+
+    repositories {
```

Comment:
pluginManagement is not declared here. Some Gradle "Magic": gradlePluginPortal implicitly added anyway and we don't need anything else yet, so let it be